packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -275,6 +281,78 @@ const gateway = new appmesh.VirtualGateway(this, 'gateway', {
   })],
   virtualGatewayName: 'gateway',
 });
+
+// A Virtual Gateway with listener TLS from a SDS provided certificate
+const gateway2 = new appmesh.VirtualGateway(this, 'gateway2', {
+  mesh: mesh,
+  listeners: [appmesh.VirtualGatewayListener.grpc({
+    port: 8080,
+    tls: {
+      mode: appmesh.TlsMode.STRICT,
+      certificate: appmesh.TlsCertificate.sds({
+        secretName: 'secrete_certificate',
+      }),
+    },
+  })],
+  virtualGatewayName: 'gateway2',
+});
+```
+
+## Adding mutual TLS authentication
+
+Mutual TLS authentication is an optional component of TLS that offers two-way peer authentication. To enable mutual TLS authentication, add `certificate` property to TLS Client Policy and/or `validation` property to TLS Listener.
```

Comment:
Please be mindful of line lengths in this file:

```suggestion
Mutual TLS authentication is an optional component of TLS that offers two-way peer authentication.
To enable mutual TLS authentication,
add `certificate` property to TLS Client Policy and/or `validation` property to TLS Listener.
```

Reply:
Well noted.

**Question:**
Should the line length be less than 150 just like `.ts` file?

Reply:
In general, yes, I would try to keep the line to below 15 characters, unless you have a good reason (an example of a good reason might be an URL that's longer than 150 characters).

Reply:
Got it. Thanks!